### PR TITLE
feat: native view managers

### PR DESCRIPTION
**Pull request by Krizzu**
# Summary

Updated templates to include examples for UI Managers in Kotlin, ObjC and Swift. View displays a box, which can change its background color by passing `color` prop (hex color).

# Test plan

1. Build the latest `bob`, generate project with `Kotlin/Obj` or `Kotlin/Swift` modules
2. Run example App, confirm the view is displayed.

## Screenshots

```react

<MyViewManager color="#32a852" style={styles.box} />

```

### Android
android

### iOS
ios

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅     |
| Android |    ✅    |

## Checklist

- [X] I have tested this on a device and a simulator

### CODE REVIEW

File: templates/swift-library/ios/{%= project.name %}ViewManager.m [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#import "React/RCTViewManager.h"
```
Comment (yvbeek) [inline templates/swift-library/ios/{%= project.name %}ViewManager.m:1]: Shoudn't this be `#import <React/RCTViewManager.h>`?

Comment (Krizzu) [reply templates/swift-library/ios/{%= project.name %}ViewManager.m:1]: I think in this case the Header search paths are the same, so both "global" and "local" import work here

**Comment by thymikee** [comment]
Great work! What do you think about adding ViewManagers as a separate command? I'm not sure how hard would it be to make it optional, @satya164 will know. But it's pretty rare for a library to include both native module and a view manager. With this change as is, users would get a lot of noise when they don't exactly need it. 

Ideally, there would be a possibility to add a view manager to the existing project bootstrapped with Bob as well, not only at the creation time.

**Comment by Krizzu** [comment]
@thymikee Thanks!

IMO if we'd plan to make `ViewManagers` optional, we should do the same for `NativeModule` to not favour one over another. This could be done by adding another inquiry after selecting module type.

**Review by satya164 — APPROVED** [review]

**Pull request merged**